types.md

```diff
@@ -202,3 +202,78 @@ TODO
 TODO
 
 ## TSR
+
+TODO
+
+## TUF
+
+Generate a TUF repository (for example, with the (Python reference implementation)[https://pypi.org/project/tuf/] or (go-tuf)[https://github.com/theupdateframework/go-tuf]). With go-tuf:
+
+```console
+$ tuf init
+$ tuf gen-key root
+$ tuf gen-key targets
+$ tuf gen-key snapshot
+$ tuf gen-key timestamp
+$ tuf add path/to/some/target.txt
+$ tuf snapshot
+$ tuf timestamp
+$ tuf commit
+```
+
+You will find the signed metadata in your TUF `repository/` directory:
+
+```console
+$ tree .
+.
+├── keys
+│   ├── snapshot.json
+│   ├── targets.json
+│   └── timestamp.json
+├── repository
+│   ├── root.json
+│   ├── snapshot.json
+│   ├── targets
+│   │   └── foo
+│   │       └── bar
+│   │           └── baz.txt
+│   ├── targets.json
+│   └── timestamp.json
+└── staged
+```
+
+Upload any TUF manifest to rekor by using the `root.json` as a the public key:
+
+```console
+$ ./rekor-cli upload --artifact repository/timestamp.json --type tuf --public-key repository/root.json
+Created entry at index 0, available at: https://rekor.sigstore.dev/api/v1/log/entries/6ed8fa5e9f0aa31b6cdfd2cc6877692f5afba52edd7ff5774eebfb22228e8847
+```
+
+View the entry with:
+
+```console
+$ rekor-cli get --uuid=31a51c1bc20da83b66b2f24899184b85dbf8261c2de8571479165619ad87cd5d
+LogID: 5c4ceffb024e0d0b50bb9e03bc308ce83a76353f1003f8e57a21c51f74cc1e0e
+Index: 0
+IntegratedTime: 2021-08-13T19:17:33Z
+UUID: 6ed8fa5e9f0aa31b6cdfd2cc6877692f5afba52edd7ff5774eebfb22228e8847
+Body: {
+  "TufObj": {
```

Comment:
Hmm, no `version` property here, is that intended?

Reply:
I'm unsure why this is, I think on get it doesn't write the version, but when I unmarshal the actual content of the stored entry I can view it.

```
asraa@asraa-glaptop:~/git/rekor$ curl -fs0  http://localhost:3000/api/v1/log/entries/68163ebd7d7ba41ece736e2910915944a85985c61892f947841729476f81b951 > logEntry.json
asraa@asraa-glaptop:~/git/rekor$ jq '."68163ebd7d7ba41ece736e2910915944a85985c61892f947841729476f81b951".body' logEntry.json | tr -d \" | base64 -d > body.json
asraa@asraa-glaptop:~/git/rekor$ cat body.json | jq
{
  "apiVersion": "0.0.1",
  "spec": {
    "metadata": {
      "content": "..."
    },
    "root": {
      "content": "..."
    },
    "spec_version": "1.0"
  },
  "kind": "tuf"
}
```